### PR TITLE
Bump version for patch release

### PR DESCRIPTION
**Pull request by jfy133**
Also fixes a mistake a noticed in the reporting of the preprint in the methods section of MQC

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,11 +3,24 @@
...
+## v1.1.3 - Augmented Akita Patch [2024-01-12]
+
+### `Added`
+
+- [#424](https://github.com/nf-core/taxprofiler/pull/424) Updated to nf-core pipeline template v2.11.1 (added by @LilyAnderssonLee & @sofstam)
 
 ### `Fixed`
 
 - [#421](https://github.com/nf-core/taxprofiler/pull/421) Updated the krakenuniq/preloadedkrakenuniq module that contained a fix for saving the output reads (❤️ to @SannaAb for reporting, fix by @Midnighter)
+- [#426](https://github.com/nf-core/taxprofiler/pull/426) Fixed preprint information information in recommended methods text (fix by @jfy133)
```
Comment (jfy133) [inline CHANGELOG.md:15]:
```suggestion
- [#427](https://github.com/nf-core/taxprofiler/pull/426) Fixed preprint information information in recommended methods text (fix by @jfy133)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,11 +3,24 @@
...
-## Unreleased
+## v1.1.3 - Augmented Akita Patch [2024-01-12]
+
+### `Added`
+
+- [#424](https://github.com/nf-core/taxprofiler/pull/424) Updated to nf-core pipeline template v2.11.1 (added by @LilyAnderssonLee & @sofstam)
 
 ### `Fixed`
 
 - [#421](https://github.com/nf-core/taxprofiler/pull/421) Updated the krakenuniq/preloadedkrakenuniq module that contained a fix for saving the output reads (❤️ to @SannaAb for reporting, fix by @Midnighter)
```
Comment (jfy133) [inline CHANGELOG.md:14]:
```suggestion
- [#419](https://github.com/nf-core/taxprofiler/pull/419) Added improved syntax highlighting for tables in documentation (fix by @mashehu)
- [#421](https://github.com/nf-core/taxprofiler/pull/421) Updated the krakenuniq/preloadedkrakenuniq module that contained a fix for saving the output reads (❤️ to @SannaAb for reporting, fix by @Midnighter)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,11 +3,25 @@
...
+
+### `Added`
+
+- [#424](https://github.com/nf-core/taxprofiler/pull/424) Updated to nf-core pipeline template v2.11.1 (added by @LilyAnderssonLee & @sofstam)
 
 ### `Fixed`
 
+- [#419](https://github.com/nf-core/taxprofiler/pull/419) Added improved syntax highlighting for tables in documentation (fix by @mashehu)
 - [#421](https://github.com/nf-core/taxprofiler/pull/421) Updated the krakenuniq/preloadedkrakenuniq module that contained a fix for saving the output reads (❤️ to @SannaAb for reporting, fix by @Midnighter)
+- [#427](https://github.com/nf-core/taxprofiler/pull/427) Fixed preprint information information in recommended methods text (fix by @jfy133)
```
Comment (LilyAnderssonLee) [inline CHANGELOG.md:16]:
```suggestion
- [#427](https://github.com/nf-core/taxprofiler/pull/427) Fixed preprint information in the recommended methods text (fix by @jfy133)
```

**Review by LilyAnderssonLee — APPROVED** [review]
Well done! You can merge after all tests have passed.

**Pull request merged**